MSCA_application.csl

```diff
@@ -0,0 +1,1256 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="never" page-range-format="chicago">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Marie Skłodowska-Curie Actions</title>
+    <title-short>Marie Curie fellowship</title-short>
+    <id>http://www.zotero.org/styles/marie-skłodowska-curie-actions</id>
+    <link href="http://www.zotero.org/styles/marie-skłodowska-curie-actions" rel="self"/>
+    <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
+    <author>
+      <name>Julian Onions</name>
+      <email>julian.onions@gmail.com</email>
+    </author>
+    <contributor>
+      <name>Elena Razlogova</name>
+      <email>elena.razlogova@gmail.com</email>
+    </contributor>
+    <contributor>
+      <name>Frank Bennett</name>
+      <email>biercenator@gmail.com</email>
+    </contributor>
+    <contributor>
+      <name>Andrew Dunning</name>
+      <email>andrew.dunning@utoronto.ca</email>
+    </contributor>
+    <contributor>
+      <name>Simon Kornblith</name>
+      <email>simon@simonster.com</email>
+    </contributor>
+    <contributor>
+      <name>Dani Bodor</name>
```

Comment:
Make yourself the author and remove the contributors.
Add a template link for the style you based this on as that is necessary to be accepted and also acts as a reference to the authors/contributors for that original style.